### PR TITLE
[Docbug][Codesamplelink update][3844]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7ede70143f04a96a6b65f7fd66a93cbf0e85af46/msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md) | 84 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/13533851-58f6-48b6-9f37-9f37dcd52d5c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md [hunk 1]
```diff
@@ -187,9 +187,9 @@ Your bot can [send proactive messages](/azure/bot-service/bot-builder-howto-proa
 
 ## Code sample
 
-| **Sample Name** | **Description** | **.NET** | **Node.js** |
-|---------------|--------------|--------|-------------|--------|
-| Proactive installation of app and sending proactive notifications | This sample shows how you can use proactive installation of app for users and send proactive notifications by calling Microsoft Graph APIs. | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-proactive-installation/csharp) | [View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/graph-proactive-installation/nodejs) |
```
Comment (Wajeed-msft) [inline msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md:192]: This is not the change which I requested. We need to add new entry in TOC for Sample code under **Utilize Teams data with Microsoft Graph** section.
image

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/665b921636b1f3cc937cfb093b741c6a8628756a/msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md) | 84 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2707337a-0363-469c-8449-587acb34cb30) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2b1e4e31939075b06fdefc79b3b0904384afef9f/msteams-platform/TOC.yml) | 96 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f2ee174e-4e95-468b-8a96-c5ec15ea3b00) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2b1e4e31939075b06fdefc79b3b0904384afef9f/msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md) | 84 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/295ba290-e429-40e6-b984-5fa9e2676972) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md (hunk 1 above)
Comment (Rajeshwari-v) [reply msteams-platform/graph-api/proactive-bots-and-messages/graph-proactive-bots-and-messages.md:192]: @Wajeed-msft, added TOC entry for code samples.

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-abirade — APPROVED** [review]
Approved

**Pull request merged**